cypress/e2e/models/migration/applicationinventory/assessment.ts

```diff
@@ -137,45 +138,49 @@ export class Assessment extends Application {
-                                    "Does the application require specific hardware?" ||
-                                $questionLine.text() ===
-                                    "How is the application clustering managed?"
-                            ) {
-                                optionToSelect = totalOptions - 1;
-                            } else {
-                                optionToSelect = Math.floor(totalOptions / 2);
-                            }
+        cy.get(assessmentBlock)
+            .its("length")
+            .then((count) => {
+                for (let i = 0; i < count - 1; i++) {
+                    cy.get(questionBlock).each(($question) => {
+                        let totalOptions = $question.find(stack).children("div").length;
+                        let optionToSelect: number;
+                        if (risk === "low") {
+                            optionToSelect = totalOptions - 1;
                             this.clickRadioOption($question, optionToSelect);
-                        });
-                } else {
-                    optionToSelect = 1;
-                    this.clickRadioOption($question, optionToSelect);
+                        } else if (risk === "medium") {
+                            cy.wrap($question)
+                                .children()
+                                .find("div.pf-v5-l-split__item")
+                                .then(($questionLine) => {
+                                    /* These 3 questions generate high risk with mean options,
+                                    hence to keep risk to medium, select last options for these set of specific questions */
+                                    if (
+                                        $questionLine.text() ===
+                                            "Does the application have legal and/or licensing requirements?" ||
+                                        $questionLine.text() ===
+                                            "Does the application require specific hardware?" ||
+                                        $questionLine.text() ===
+                                            "How is the application clustering managed?"
+                                    ) {
+                                        optionToSelect = totalOptions - 1;
+                                    } else {
+                                        optionToSelect = Math.floor(totalOptions / 2);
+                                    }
+                                    this.clickRadioOption($question, optionToSelect);
+                                });
+                        } else {
+                            optionToSelect = 1;
+                            this.clickRadioOption($question, optionToSelect);
+                        }
+                    });
+                    if (i === 4) {
+                        clickJs(commonView.nextButton);
```

Comment:
I guess we have `next()` function in `utils` for such purpose

Reply:
@ibragins 
I don't understand why these functions are present, what's the purpose of them, also the else statement executes the same command i.e. `clickJs(commonView.nextButton);` I didn't remove them because I didn't want anything to affected by it